docker/py3-native/README.md

```diff
@@ -3,7 +3,32 @@
 This README purpose is to clarify the following:
 * Which docker images the native image supports
 * The purpose of the **verifiers** folder
-* OS dependencies for each docker image that the native image supports.
+* OS dependencies for each docker image that the native image supports
+* What are native images and how to handle them
+
+
+## How To Handle Native Images
+### What is a native image?
+* Native image is a combination of a lot of docker images that scripts and integrations use. 
+* The native image contains all the OS / python dependancies that are used in the supported docker images
+* For example, given the docker image taxii2, tesseract and chromium, the native image will contain all of their OS / python dependencies. 
+* That means that every integration/script that uses taxii2, tesseract or chromium docker image can also be used in the native image and there is compatibility between them.
+
+### What should I do when changing a docker-image in the dockerfiles repo?
+* Check if the docker image that was changed is supported also in the native image. It is possible to check it in the docker native image configuration file. Note: if you have changed a docker-image that is supported in the native image the native docker validator workflow will fail and alert you. 
+* if the docker image is supported in the native image, make sure that the changes that you made to that docker image will also be applied in the native image.
+* If a new python dependency was added to the docker image, make sure it's also added to the native image.  
```

Comment:
add these points as a sublist (indent it)

Reply:
what do you mean by "sublist" - and the points are already indented by markdown

Reply:
If the docker image is supported by the native image, apply the same changes to the native image. <- this is general
 
  If a new python dependency was added to the docker image, make sure it's also added to the native image. <- This is an example of that. So I mean have it like 
  
Main List
Point 1 
Point 2
    Example of point 2
    Another Example
Point 3

Reply:
done + proper examples